portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,35 @@
 package com.google.sps.servlets;
 
+
 import java.io.IOException;
+import java.util.ArrayList;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
```

Comment:
Let's rename the endpoint from `/hello` to something more descriptive?  Having a good API naming convention makes it easier to understand and work with in the long run - See https://restfulapi.net/resource-naming/ for bonus reading.